src/netaddress.h

```diff
@@ -80,6 +80,11 @@ class CNetAddr
         bool GetInAddr(struct in_addr* pipv4Addr) const;
         uint32_t GetNetClass() const;
 
+        //! For IPv4, mapped IPv4, SIIT translated IPv4, Teredo, 6to4 tunneled addresses, return the relevant IPv4 address as a uint32.
+        bool HasLinkedIPv4() const;
+        //! Whether this address has a linked IPv4 address (see GetLinkedIPv4()).
```

Comment:
This comment should be on L83 and vice versa? :)

Reply:
Fixed.